pkg/metricscollector/v1beta1/file-metricscollector/file-metricscollector.go

```diff
@@ -52,7 +65,7 @@ func CollectObservationLog(fileName string, metrics []string, filters []string,
 	case commonv1beta1.JsonFormat:
 		return parseLogsInJsonFormat(strings.Split(logs, "\n"), metrics)
 	}
-	return nil, fmt.Errorf("format must be set %v or %v", commonv1beta1.TextFormat, commonv1beta1.JsonFormat)
+	return nil, nil
```

Comment:
```suggestion
```

Reply:
That makes sense. But it seems that the control flow won't reach them. Do we need to add this line for the neatness of the code?

Reply:
I meant to remove this line.

Reply:
Sorry. I meant the 'default' line.